src/helpers/Location.ts

```diff
@@ -8,35 +8,37 @@
  *
  * class Location extends URL {
  *   get url() {
- *     return this.pathname + this.search;
+ *	 return this.pathname + this.search;
  *   }
  * }
  *
  */
 
-export default class Location {
-	constructor(url, base = document.baseURI) {
-		const location = new URL(url.toString(), base);
-		Object.defineProperty(location, 'url', { get: () => location.pathname + location.search });
-		return location;
+export default class Location extends URL {
+	constructor(url: string, base: string = document.baseURI) {
+		super(url.toString(), base);
+	}
+
+	get url() {
+		return this.pathname + this.search;
 	}
```

Comment:
Pretty much only place with updated logic, since couldn't figure out what `defineProperty` does to the original one.

Reply:
Aha, I see! I was wondering about that.  
Hm that's a tough one... Can we setup the target browsers in a way that we won't need the polyfil (and babel won't add it), or is it still required?

Reply:
Yes, fiddling with the babel target is the way to go here. I'll give it a try!

Reply:
I've tested a browserslist config that gets rid of 1K of polyfills by skipping Safari 12 🤡 WIll create a PR.